src/store/Workspaces/index.ts

```diff
@@ -299,7 +300,24 @@ export const actionCreators: ActionCreators = {
       dispatch({ type: 'DELETE_WORKSPACE', workspaceId });
     } catch (e) {
       dispatch({ type: 'RECEIVE_ERROR' });
-      throw new Error(`Failed to delete the workspace, ID: ${workspaceId}, ` + e);
+
```

Comment:
It is a common problem for our error messages. I will prefer to implement this logic as a separate function and move it to the separate file in src/services/helpers.

WDYT?

Reply:
I think this should be addressed as a separate issue.